tests/testExamples.m

```diff
@@ -1,61 +1,89 @@
         exampleFile = {'orsoDSPCStandardLayers',...
             'DPPCCustomXYScript',...
             'testCppCustomLayers',...
             'DSPCMonolayerStandardLayers',...
-            'DSPCMonolayerTwoContrasts'};
+            'DSPCMonolayerTwoContrasts',...
+            'orsoDSPCCustomLayers'};
         exampleName = {'Orso lipid example',...
             'Custom DPPC',...
             'Orso lipid example - custom layers',...
             'DSPC monolayers',...
-            'DSPC monolayers'};
+            'DSPC monolayers',...
+            'Orso lipid example - custom layers'};
         exampleModel = {modelTypes.StandardLayers.value,...
             modelTypes.CustomXY.value,...
             modelTypes.CustomLayers.value,...
             modelTypes.StandardLayers.value,...
-            modelTypes.StandardLayers.value};
+            modelTypes.StandardLayers.value,...
+            modelTypes.CustomLayers.value};
         exampleContrastCount = {3,...
             3,...
             3,...
             4,...
-            2};
+            2,...
+            3};
         exampleLayerCount = {5,...
             0,...
             0,...
             4,...
-            4};
+            4,...
+            0};
         exampleGeometry = {geometryOptions.AirSubstrate.value,...
             geometryOptions.SubstrateLiquid.value,...
             geometryOptions.SubstrateLiquid.value,...
             geometryOptions.AirSubstrate.value,...
-            geometryOptions.AirSubstrate.value};
+            geometryOptions.AirSubstrate.value,...
+            geometryOptions.SubstrateLiquid.value};
         exampleParameterParamCount = {15,...
             6,...
             8,...
             10,...
-            10};
+            10,...
+            8};
         exampleBulkOutParamCount = {3,...
             3,...
             3,...
             2,...
-            2};
+            2,...
+            3};
         exampleBackgroundsTypesCount = {3,...
             3,...
             3,...
             2,...
-            2};
+            2,...
+            3};
         exampleDataCount = {4,...
             1,...
             4,...
             6,...
-            3};
+            3,...
+            4};
 
     end
 
-    methods (TestMethodTeardown)
+    methods(TestClassSetup)
+        function addExamplePaths(testCase)
+            root = getappdata(0,'root');
+            for i = 1:length(testCase.exampleFolder)
+                addpath(fullfile(root, 'examples', testCase.exampleFolder{1,i}));
+            end
+        end
+    end
+
+    methods(TestClassTeardown)
         function testCleanUp(testCase)
+            root = getappdata(0,'root');
+            for i = 1:length(testCase.exampleFolder)
+                rmpath(fullfile(root, 'examples', testCase.exampleFolder{1,i}));
+            end
+            % Generated by the DSPCMonolayerTwoContrasts example
             if exist('twoContrastExample.mat', 'file') == 2
```

Comment:
This deletes all instances of twoConstrastExample.mat from the repo including the ones in the examples folder. Please remove this and replace with 

```
    methods (TestClassSetup)
        function setWorkingFolder(testCase)
            % Makes a temporary folder the current working directory so 
            % file written by the example are deleted 
            import matlab.unittest.fixtures.WorkingFolderFixture;
            
            testCase.applyFixture(WorkingFolderFixture);
        end
    end
```